xmodule/library_content_module.py

```diff
@@ -433,8 +434,9 @@ def author_view(self, context):
         fragment = Fragment()
         root_xblock = context.get('root_xblock')
         is_root = root_xblock and root_xblock.location == self.location
-
-        if is_root:
+        task_status = self.tools.import_task_status(self)
+        is_loading = task_status == 'In Progress'
```

Comment:
There is no variable containing `'In Progress'` status in the module?

Reply:
updated